src/controllers/RecommendationController.js

```diff
@@ -0,0 +1,177 @@
+import { Recommendation } from '../models';
+import { statusCodes, responseMessages } from '../constants';
+
+/**
+ * @description Re Controller class
+ */
+export default class RecommendationController {
+
+    /**
+     * Recommend a user
+     *
+     * @static
+     * @param {*} req
+     * @param {*} res
+     * @returns {object} recommendation
+     * @memberof RecommendationController
+     */
+    static async recommend(req, res) {
```

Comment:
@gracelungu Can you use the same naming as for other controllers to keep conformity? `createRecommendation`